src/rfc3986/abnf_regexp.py

```diff
@@ -71,15 +71,15 @@
 
 # Host patterns, see: http://tools.ietf.org/html/rfc3986#section-3.2.2
 # The pattern for a regular name, e.g.,  www.google.com, api.github.com
-REGULAR_NAME_RE = REG_NAME = '((?:{0}|[{1}])*)'.format(
```

Comment:
Unnamed groups are way faster than named groups. We have a lot of groups that shouldn't be named so I replaced all that I could find with unnamed groups. I can revert this change if desired.